browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
```

Comment:
This is another one I dislike because AFAIK only warns against purely aesthetic cases, where parentheses tend to confuse the reader because they now need to puzzle out if you put the parens for aesthetic purposes or for some weird case. I generally find it's less ambiguous to just omit them where they're not necessary, but if you feel strongly we can change it everywhere by moving it to the base config level to keep a consistent style